packages/mobile/src/components/components.ts

```diff
@@ -28,3 +29,4 @@ export { FlagBackgroundView } from './flag-background-view/flag-background-view'
 export { PrimaryButton } from './primary-button/primary-button';
 export { SecondaryButton } from './secondary-button/secondary-button';
 export { BuildInfo } from './build-info/build-info';
+export * from 'react-native-popover-view';
```

Comment:
```suggestion
export { default as Popover } from 'react-native-popover-view';
```

...and if you need types re-export them within `common/types`